diagram.md

```diff
@@ -208,11 +208,11 @@ Below is a representation of the Block structure.
+The `Timecode Element` value MUST be stored once per `Cluster`. Another `Timecode` SHOULD be stored within the `Block` structure itself. The `Timecode` in the `Cluster` is relative to the entire `Segment`. The `Timecode` SHOULD be the first `Block` in the `Cluster`. The `Timecode` in the `Block` itself is relative to the `Timecode` in the `Cluster`. For example, if the `Timecode` in the `Cluster` is set to 10 seconds and a `Block` in that `Cluster` is supposed to be played 12 seconds into the clip, the `Timecode` in the `Block` would be set to 2 seconds.
 
-The `ReferenceBlock` in the BlockGroup, is used instead of the basic "P-frame"/"B-frame" description. Instead of simply saying that this Block depends on the Block directly before, or directly afterwards, we put the timecode of the needed Block. And because you can have as many `ReferenceBlock Elements` as you want for a Block, it allows for some extremely complex referencing.
+The `ReferenceBlock` in the `BlockGroup` is used instead of the basic "P-frame"/"B-frame" description. Instead of simply saying that this `Block` depends on the `Block` directly before, or directly afterwards, the `Timecode` of the necessary `Block` is used. Because there can be as many `ReferenceBlock Elements` as necessary for a `Block`, it allows for some extremely complex referencing.
 
-The `Cues Element` is used to seek when playing back a file by providing a temporal index for each of the tracks. It is similar to the `SeekHead Element`, but this is used for seeking to a specific time when playing back the file. Without this it is possible to seek, but it is much more difficult because the player has to 'hunt and peck' through the file looking for the correct timecode. `Cues` contains `CuePoint Elements` which store the timecode (`CueTime`) and then a listing for the exact position in the file for each of the tracks for that timecode. The `Cues` are pretty flexible for what exactly you want to index. For instance, you can index every single timecode of every `Block` or index selectively. If you have a video file, it is RECOMMENDED to index at least the keyframes of the video track.
+The `Cues Element` is used to seek when playing back a file by providing a temporal index for each of the `Tracks`. It is similar to the `SeekHead Element`, but used for seeking to a specific time when playing back the file. It is possible to seek without this element, but it is much more difficult because the player has to 'hunt and peck' through the file looking for the correct timecode. `Cues` MAY contain `CuePoint Elements` which store the timecode (`CueTime`) and a listing for the exact position in the file for each of the tracks for that timecode. The `Cues` are flexible. For instance, you can index every single timecode of every `Block` or index selectively. For video files, it is RECOMMENDED to index at least the keyframes of the video track.
```

Comment:
> `Cues` MAY contain `CuePoint Elements`…

I'd prefer SHOULD here as I cannot think of any use case where empty `Cues` would be used. MUST would be too strong as I wouldn't want to invalidate a file just because there's an empty `Cues` master present.